packages/api/app/Controllers/Http/TechnologyController.js

```diff
@@ -618,6 +657,18 @@ class TechnologyController {
 			.where({ status: questionStatuses.ANSWERED })
 			.withParams(request, { filterById: false });
 	}
+
+	async getRevisionHistory({ params }) {
+		const technology = await Technology.query()
+			.getTechnology(params.id)
+			.with('comments')
+			.with('revisions.reviewer.user')
```

Comment:
```suggestion
			.with('revisions.reviewer')
```
Acho que podemos usar só o **user_id** que vem no reviewer